pkg/monitor/monitor/util.go

```diff
@@ -491,6 +491,25 @@ func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.T
 		commands = append(commands, "--storage.tsdb.max-block-duration=2h")
 		commands = append(commands, "--storage.tsdb.min-block-duration=2h")
 	}
+
+	//add readiness probe
+	var readinessProbeHandler core.Handler
+	{
+		readyPath := "/-/ready"
+		readinessProbeHandler.HTTPGet = &core.HTTPGetAction{
+			Path: readyPath,
+			Port: intstr.FromInt(9090),
+		}
+
+	}
+	readinessProbe := &core.Probe{
```

Comment:
Please also add livenessProbe.

Reply:
And what about also adding the probes for the Grafana.

Reply:
I don’t recommend adding LivenessProbe, you can refer to https://github.com/prometheus-operator/prometheus-operator/pull/3502.

Reply:
OK.